internal/scaffold/role.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
 
 	log "github.com/sirupsen/logrus"
-	yaml "gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v3"
```

Comment:
If we switch this to `sigs.k8s.io/yaml`, we might be able to simplify [the updateRoleFile function](https://github.com/operator-framework/operator-sdk/pull/2786/files#diff-14b22f7ce65c82647940d5ce91460f9aR146-R164) by removing the initial `json.Marshal` and `yaml.Unmarshal` steps

Reply:
done